### PR TITLE
Add new `once` combinator

### LINKED ISSUE #74: Consider adding a `once` primitive
We had a discussion about this over on [StackOverflow](https://stackoverflow.com/questions/27816012/performing-a-single-switch-in-reactive-banana/27878929?noredirect=1#comment44315088_27878929).

### PR DESCRIPTION
**Pull request by ocharles**
Fixes #74

### CODE REVIEW

**Review by mitchellwrosen — COMMENTED** [review]
LGTM, but shouldn't it be in MonadMoment?

**Comment by ocharles** [comment]
Oh yes, good catch!

**Review by mitchellwrosen — APPROVED** [review]

File: reactive-banana/src/Reactive/Banana/Combinators.hs [hunk 1]
```diff
@@ -349,6 +350,15 @@ split e = (filterJust $ fromLeft <$> e, filterJust $ fromRight <$> e)
     fromRight (Right b) = Just b
 
 
+-- | Keep only the next occurence of an event.
+--
```
Comment (HeinrichApfelmus) [inline reactive-banana/src/Reactive/Banana/Combinators.hs:354]:
```suggestion
-- (Also aids the garbage collector by indicating that the result event can be discarded after its only occurrence.)
```

**Review by HeinrichApfelmus — APPROVED** [review]
Looks great, thanks! Could you commit my suggestion concerning garbage collection? The it's ready to go!

File: reactive-banana/src/Reactive/Banana/Combinators.hs (hunk 1 above)
Comment (ocharles) [reply reactive-banana/src/Reactive/Banana/Combinators.hs:354]: Sounds good! I'm going to slightly reword this without parens, as it seems important enough to stand alone.

**Pull request merged**